src/main/native/windows/processes-jni.cc

```diff
@@ -352,6 +352,13 @@ Java_com_google_devtools_build_lib_windows_jni_WindowsProcesses_nativeCreateProc
           err_code);
       return PtrAsJlong(result);
     }
+    if (!SetFilePointerEx(stdout_process, {0}, NULL, FILE_END)) {
```

Comment:
Is this the intended behavior of this API?

Reply:
What do you mean?

Reply:
Ignore, I misread the code.